hexrd/ui/llnl_import_tool_dialog.py

```diff
@@ -138,6 +141,11 @@ def load_config(self):
         self.config_file = selected_file if selected_file else None
         self.ui.config_file_label.setText(os.path.basename(self.config_file))
         self.ui.config_file_label.setToolTip(self.config_file)
+        if self.ui.instrument.isEnabled():
+            # Only set the instrument config once
+            success = self.get_instrument_defaults()
+            if not success:
+                return
```

Comment:
Is there supposed to be different behavior depending on the value of `success`? It looks like this function returns anyways after this block ends.

Reply:
Nope, careless copy-paste. Fixed.